### ISSUE TITLE
🐛 BUG: Dynamic labels

### ISSUE THREAD

**Issue by jurjendevries** [dilmaheu/dilmahtea.me#175]
**Describe the bug**
There are many labels that should be dynamic for accessible elements. E.g. at checkout there is a button 'Change' on the shipping page. For a screen reader / user with eye illness disability its not possible to understand the word 'Change' without any context that it has to do with changing the shipping address details.

**Steps to Reproduce**
1. Go to Checkout shipping
2. See the change button
4. Read it with a screen reader and find out it only reads 'Change'

**Expected behavior**
In this sample there should be a CMS field for such a label and for every language were we can extend it with the context such as 'Change the shipping details' (note that this should not be the button text itself, just the label)

**Screenshots**
N/A

**Desktop (please complete the following information):**
Desktop independent

**Smartphone (please complete the following information):**
Smartphone independent

**Additional context**
It was originally from https://github.com/dilmaheu/dilmahtea.me/pull/168 but because there are many labels @RafidMuhymin and I suggested to have the research of which labels needs this and code will be done by @abir
